diktat-rules/src/test/kotlin/com/saveourtool/diktat/util/FixTestBase.kt

```diff
@@ -56,6 +56,14 @@ open class FixTestBase(
         val testComparatorUnit = testComparatorUnitSupplier(overrideRulesConfigList)
         val result = testComparatorUnit
             .compareFilesFromResources(expectedPath, testPath, resourceReader)
+        if (!result.isSuccessful) {
```

Comment:
It was not enough information, like it was before?

Reply:
yes, it returns an useful diff in IDEA when JUnit compares Strings